erpnext/stock/doctype/delivery_trip/delivery_trip.py

```diff
@@ -158,8 +158,11 @@ def form_route_list(self, optimize):
 			(list of list of str): List of address routes split at locks, if optimize is `True`
 		"""
 
-		settings = frappe.get_single("Google Maps Settings")
-		home_address = get_address_display(frappe.get_doc("Address", settings.home_address).as_dict())
+		settings = frappe.get_doc("Google Maps", {"entity": self.driver})
```

Comment:
If the `Google Maps` document doesn't exist for the driver, `get_doc` will raise a `DoesNotExistError` exception and fail.